modules/programs/ssh.nix

```
@@ -181,6 +231,9 @@ let
     ++ optional (cf.proxyCommand != null)    "  ProxyCommand ${cf.proxyCommand}"
     ++ optional (cf.proxyJump != null)       "  ProxyJump ${cf.proxyJump}"
     ++ map (file: "  IdentityFile ${file}") cf.identityFile
+    ++ map (f:
+      "  LocalForward [${f.bindAddress}]:${toString f.port} [${f.host}]:${toString f.hostPort}"
```

Maybe split over multiple lines?
```suggestion
      "  LocalForward"
      + " [${f.bindAddress}]:${toString f.port}"
      + " [${f.host}]:${toString f.hostPort}";
```